scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -17,28 +17,6 @@ def test_dispatch_to_unrecognized_library():
     xp_assert_close(res, ref)
 
 
-@pytest.mark.parametrize('dtype', ['float32', 'float64', 'int64'])
-def test_rel_entr_generic(dtype):
```

Comment:
I understand that float32 and float64 is redundant with `test_support_alternative_backends` just below.
int64 is improper as discussed above.